files/image_config/monit/memory_checker

```diff
@@ -128,6 +133,10 @@ def main():
     parser.add_argument("threshold_value", type=int, help="threshold value in bytes")
     args = parser.parse_args()
 
+    if not is_service_active("docker"):
+        syslog.syslog(syslog.LOG_INFO,"Skipping memory check: docker daemon is down")
```

Comment:
Can we rephrase the syslog message as following?

`syslog.syslog(syslog.LOG_INFO, "[memory_checker] Exits without checking memory usage of container '{}' since docker daemon is not running!".format(args.container_name))`

Reply:
Done